src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_3_layout.html.twig

```diff
@@ -203,6 +203,18 @@
     {{- parent() -}}
 {%- endblock translate_fields_widget %}
 
+{% block datePicker_widget %}
```

Comment:
it's aesthetic but I'd prefer something like ``date_picker``, what do you think ?